scripts/test.sh

```diff
@@ -81,26 +102,118 @@ start_packager() {
 }
 
 xctest() {
-  local dest="$(xcrun simctl list devices | grep -v unavailable | grep -m 1 -o '[0-9A-F\-]\{36\}')"
+  # note: the DEVELOPER_DIR and IOS_SIM_DEVICE sections sould be kept in sync with setup_ios_simulator
```

Comment:
ahhh, so it's not that `DEVELOPER_DIR` needs to be synced with `IOS_SIM_DEVICE`, but rather both places where they're set (Jenkins vs non-Jenkins), if you change one you should change the other? In any case, another reason why an explanatory comment is useful here.